database-restore-n8n.sh

```
@@ -18,7 +18,8 @@ if [[ -f "$dir/.backup" ]]; then
 fi
 
 # Set backup directory
-backup_dir="${BACKUP_DIR:-$dir/db-backup/n8n}" 
+base_dir="${RESTORE_DIR:-$dir/db-restore"  # Default to $dir/db-backup if RESTORE_DIR is not set
```

Shellcheck (0.10.0)

[info] The mentioned syntax error was in this parameter expansion.

(SC1009)